src/main/java/com/jcabi/github/RtUser.java

```diff
@@ -158,7 +160,16 @@ public UserEmails emails() {
 
     @Override
     public void markAsRead(final Date lastread) throws IOException {
```

Comment:
@llorllale looks nice. Can you leave here a puzzle to start with ``User.notifications()`` as you suggested [here](https://github.com/jcabi/jcabi-github/issues/1304#issuecomment-378642497)? Sorry, by the way, I completely forgot about it, even after I'd seen your ping :(